1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 52 Line 1_
```markdown
Python strings can be “formatted” with the format method: 
```
Unless you feel strongly, I'd prefer to use the newest of Python's many string formatting methods, f-strings

```
print(f"The area of a circle of radius {r} is {math.pi * math.pow(r, 2)}")
```